user-interface/src/main/java/life/qbic/datamanager/views/projects/project/samples/SampleInformationMain.java

```diff
@@ -94,9 +99,10 @@ public class SampleInformationMain extends Main implements BeforeEnterObserver {
   private final transient CancelConfirmationDialogFactory cancelConfirmationDialogFactory;
   private final transient MessageSourceNotificationFactory notificationFactory;
   private final transient SampleValidationService sampleValidationService;
-  private final transient TemplateService templateService;
   private final transient SampleRegistrationServiceV2 sampleRegistrationServiceV2;
   private final transient AsyncProjectService asyncProjectService;
+  private final MessageSourceNotificationFactory messageFactory;
+  private Toast pendingTask;
```

Comment:
What happens if multiple tasks are started? The first task would then remove the toast disregarding other ongoing tasks in the background. Your assumption that there is one operation at a time and that the user waits for that to finish should go into the code (as comment for example) if it is desired.

Reply:
I agree, I will work on a solution